tutor/commands/compose.py

```diff
@@ -33,9 +35,12 @@ def docker_compose(self, *command: str) -> int:
         """
         Run docker-compose with the right yml files.
         """
-        if "start" in command or "up" in command or "restart" in command:
-            # Note that we don't trigger the action on "run". That's because we
-            # don't want to trigger the action for every initialization script.
+        # Trigger the action just once per runtime
+        start_commands = ("start", "up", "restart", "run")
+        if not ComposeTaskRunner.HOOK_FIRED and any(
```

Comment:
This should be `self.HOOK_FIRED`, and not `ComposeTaskRunner.HOOK_FIRED`. Same below.

Reply:
Great catch! I didn't know that.